module-2/backend/week_one.md

```diff
@@ -17,16 +33,58 @@ Note: When you're done, submit a PR.
+    @count = 1
+    render locals: { name : 'Mr. Ed' }
+    erb :index
+  end
+  ```
+
 8. In the same code block, how would I pass a local variable `name` with a value of `Mr. Ed` to the view?
+  - See above.
+
 9. What's the purpose of ERB?
+  - The purpose of Embedded Ruby(ERB) is to incorporate Ruby method attributes and behaviors into HTML text, allowing one to display ruby method values onto an applications view page. 
+
 10. Why do I need a development AND test database?
+  - It is necessary to have both a development database for actual program implementation, and another exclusively for testing. Everytime tests are run, one would want to start with a fresh slate with not existing data in the test database. By having two separate databases, there is still access to all existing data when shotgun is run and the app is viewed within the browser.
+
 11. What is CRUD and why is it important?
+  - Create Read Update Destroy (CRUD)
+  - CRUD is a list of operations that can be performed on a resource. In this context, resource is a generic term for something that exists in the application. Generally, each table created in the database hodls specific instances of a particular resource. 
+
 12. What does HTTP stand for? 
+  - HyperText Transfer Protocol
+
 13. What are the two ways to interpolate Ruby in an ERB view template? What's the difference between these two ways?
+  - Two ways to interpolate Ruby in an ERB view template are:
+      - <%muffins%> (utilized when you are interpolating ruby behavior)
+      - <%=muffins%> (utilized when you are interpolating ruby behavior, renders the value of the returned statement)
+
 14. What's an ORM?
+  - Object Relational Mapping(ORM), is the means of making incompatible information applicable through written code.
+
 15. What's the most commonly used ORM in ruby (Sinatra & Rails)?
+  - ActiveRecord
+
 16. Let's say we have an application with restaurants. There are seven verb + path combinations necessary to provide full CRUD functionality for our restaurant application. List each of the seven combinations, and explain what each is for.
+  - What a user wants
+  - Method(verb)
+  - URI(Uniform Resource Identifier)
+  - Data Prep/Manipulation
+  - Redirect or Render
+  - Name of View
+
 17. What's a migration? 
+  - A migration is when file information is passed form a file into a database defined by specific instructions provided within the migration file.
+
 18. When you create a migration, does it automatically modify your database?
+  - When you create a migration, it does not automatically modify your database. Rather, it creates a migration file containing an empty 'def change' method which you would then use to 'format' how you want to pull information out the file you intend to use as the source for your databse.
+
 19. How does a model relate to a database?
+  - A model relates to a database by taking additional information (has_many and belongs_to) in order to define specific relationships.

```

Comment:
This sounds right. I believe we were thinking along the lines of a model corresponding to a **table** in the database, with an instance of that model corresponding to a **row** in that table.